tools/v8_gypfiles/v8.gyp

```diff
@@ -1378,6 +1378,9 @@
           '<(V8_ROOT)/src/heap/base/worklist.h',
         ],
         'conditions': [
+          ['enable_lto=="true"', {
```

Comment:
IIUC this would also apply when it's built with clang? Can we make this gcc only?

Reply:
Already implied by `enable_lto=="true"`.

https://github.com/nodejs/node/blob/4243ce03182aea72aefdd2ff29ecc1affbc59716/configure.py#L168-L171

https://github.com/nodejs/node/blob/4243ce03182aea72aefdd2ff29ecc1affbc59716/configure.py#L1219-L1226

Reply:
By the way, we should allow LTO with Clang... but that's for another change.

Reply:
I see, didn't realized that we have this guard here..yeah enabling it for clang sounds good but I agree that's a separate issue.